src/utils/random-data.js

```diff
@@ -182,6 +182,7 @@ class Pipeline {
    * @param {number} initialRank Rank index
    * @returns {object} Node object
    */
+  /* eslint-disable camelcase */
```

Comment:
see my above comment - I would suggest adding an extra line here to remain consistent with the other files.

Furthermore, is this disabled rule only intended to be applied to `createNode`, or the rest of the file from line 186 onwards? If it's the former, we will also need to add in `/* eslint-disable camelcase */` at the end of the function to ensure it does not affect the rest of the file.